lib/index.js

```diff
@@ -148,25 +148,25 @@ module.exports = function (opts) {
     let pkgInfo = null;
 
     opts = getOptions(opts);
-
-    return pkgd()
-        .then(info => {
-
-            pkgInfo = info;
-
-            if (opts.prepublishScript)
-                return runPrepublishScript(opts.prepublishScript);
-        })
-        .then(() => runValidations(opts, pkgInfo))
-        .then(() => {
-            if (!module.exports.testMode)
-                return printReleaseInfo(pkgInfo.cfg.version, opts.tag);
-        })
-        .then(() => opts.confirm ? confirmPublish() : true)
-        .then(ok => {
-            if (ok)
-                return publish(opts.tag);
-        });
+    (opts.prepublishScript
```

Comment:
It can be done this way:
```js
return Promise.resolve()
    .then(() => {
        if (opts.prepublishScript)
           return runPrepublishScript(opts.prepublishScript);      
    })
    .then(pkgd)
    ....
```